
This patch addresses the following clang failure: tayga-0.9.2/nat64.c:119:6:
error: call to undeclared function 'writev'; ISO C99 and later do not support
implicit function declarations [-Wimplicit-function-declaration]

Upstream-Status: Pending
Signed-off-by: Pawel Langowski <pawel.langowski@3mdeb.com>

--- tayga.h	2023-09-29 14:38:05.005837695 +0200
+++ tayga-2.h	2023-09-29 14:37:45.560837257 +0200
@@ -20,6 +20,7 @@
 #include <sys/stat.h>
 #include <sys/ioctl.h>
 #include <sys/socket.h>
+#include <sys/uio.h>
 #include <netinet/in.h>
 #include <arpa/inet.h>
 #include <unistd.h>
